spec/helpers/shopping_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the ShoppingHelper. For example:
+#
+# describe ShoppingHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
[REQUIRED] As suggested by the previous reviewer kindly remove all commented-out unused code.